docs/getting-started/quick-start.md

```diff
@@ -32,43 +32,49 @@ psql -h 127.0.0.1 -p 4003 -d public
 
 ## Create tables
 
-Suppose you have a table named `grpc_latencies` that stores the gRPC latencies of your application.
+Suppose you have a wide event table named `grpc_latencies` that stores the gRPC latencies of your application.
 The table schema is as follows:
 
 ```sql
 CREATE TABLE grpc_latencies (
   ts TIMESTAMP TIME INDEX,
-  host STRING,
+  host STRING INVERTED INDEX,
   method_name STRING,
   latency DOUBLE,
   PRIMARY KEY (host, method_name)
 ) with('append_mode'='true');
 ```
 
 - `ts`: The timestamp when the metric was collected. It is the time index column.
-- `host`: The hostname of the application server. It is a tag column.
-- `method_name`: The name of the RPC request method. It is a tag column.
+- `host`: The hostname of the application server, enabling [inverted index](/user-guide/manage-data/data-index#inverted-index).
+- `method_name`: The name of the RPC request method.
 - `latency`: The latency of the RPC request.
 
+And it's append only.
+
 Additionally, there is a table `app_logs` for storing application logs:
 
 ```sql
 CREATE TABLE app_logs (
   ts TIMESTAMP TIME INDEX,
-  host STRING,
+  host STRING INVERTED INDEX,
   api_path STRING,
   log_level STRING,
-  `log` STRING,
+  log_msg STRING FULLTEXT INDEX WITH('case_sensitive' = 'false'),
   PRIMARY KEY (host, log_level)
 ) with('append_mode'='true');
 ```
 
 - `ts`: The timestamp of the log entry. It is the time index column.
-- `host`: The hostname of the application server. It is a tag column.
+- `host`: The hostname of the application server, enabling inverted index.
 - `api_path`: The API path.
-- `log_level`: The log level of the log entry. It is a tag column.
-- `log`: The log message.
+- `log_level`: The log level of the log entry.
+- `log_msg`: The log message, enabling [fulltext index](/user-guide/manage-data/data-index#fulltext-index).
 
+ It's append only, too.
```

Comment:
```suggestion
- Set `append_mode` to true to preserve duplicate rows.
```

Reply:
To preserve duplicate rows is not the purpose but the side effect. Append only is for performance.